modules/images-update-global-pull-secret.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+// * openshift_images/using-image-pull-secrets
+
+[id="images-update-global-pull-secret_{context}"]
+= Updating the global cluster pull secret
+
+You can update the global pull secret for your cluster.
+
+[WARNING]
+====
+Modifying the global cluster pull secret can potentially render the cluster unavailable.
+====
```

Comment:
Ominous statement. Do we have any information on what will make the cluster unavailable and what to do about it?

Reply:
@smarterclayton You mentioned in the email to call this out - do you have more info that we can provide as @mburke5678 suggests?

Preview:
https://osdocs-660-insights--ocpdocs.netlify.com/openshift-enterprise/latest/support/insights-operator/disabling-insights-operator.html#images-update-global-pull-secret_insights-operator-disabling

Reply:
How about 
`cluster resources need to self-adjust to new pull secret which can temporary limit the usability of the cluster` .. it's less threatening ..

Reply:
:+1: for not scaring people off

Reply:
I think it's still too threatening ;).  What usability limits do we expect?  It will take a bit to get the new secret rolled out, and we currently have some hiccups around availability during reboots ([rhbz#1845411][1] and [rhbz#1809665][2]).  If that's the expected usability impact, I'd rather call it out explicitly.  As it stands, it seems to be stirring up fear and uncertainty without providing the reader with a means to assess the trade-offs (or importance of bumping pull secrets during a scheduled maintenance window, or things to watch for during that maintenance window, or how to judge completion).

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1845411
[2]: https://bugzilla.redhat.com/show_bug.cgi?id=1809665

Reply:
@wking This is an old PR that has already been merged. If you think we need to modify this wording, would you mind opening a BZ or GitHub issue so that we can properly track and discuss it? Thanks!

Reply:
Spun off into #23378.